frontend/globals/unity-util.ts

```diff
@@ -196,6 +196,37 @@ export class UnityUtil {
 		return UnityUtil.onReady();
 	}
 
+	/**
+	 * Load an Unreal PixelStreaming Viewer into the provided Div. The PixelStreaming Viewer is a class
+	 * that manages a WebRtc PeerConnection to a remote Unreal instance via a video stream & data channel.
+ 	 * @category Configurations
+	 * @param div - the html dom element that the viewer should be created under
+	 * @param config - an object describing the configuration of the viewer
+	 * @return returns a promise which resolves when the game is loaded.
+	 */
+	public static loadUnreal(div: any, config: any): Promise<void> {
```

Comment:
instead of removing the class declaration and make config: any, i think we can use interface?